torch_object_detection_segmenter.py

```diff
@@ -19,6 +19,7 @@ class TorchObjectDetectionSegmenter(Executor):
     from an image using `torchvision detection models`
     and crops the images according tothe detected bounding boxes
     of the objects with a confidence higher than a threshold.
+    :param on_gpu: set to True if using GPU
```

Comment:
if u want to use gpu, will only affect if GPU is available

Reply:
users will only use gpu if it's available right